nck/readers/dv360_reader.py

```diff
@@ -147,35 +138,53 @@ def get_sdf_body(self):
         return {
             "parentEntityFilter": {
                 "fileType": self.kwargs.get("file_type"),
-                "filterType": self.kwargs.get("filter_type")
+                "filterType": self.kwargs.get("filter_type"),
             },
             "version": self.SDF_VERSION,
-            "advertiserId": self.kwargs.get("advertiser_id")
+            "advertiserId": self.kwargs.get("advertiser_id"),
         }
 
     def get_sdf_objects(self):
         body = self.get_sdf_body()
         init_operation = self.create_sdf_task(body=body)
         created_operation = self._wait_sdf_download_request(init_operation)
         if "error" in created_operation:
-            raise SdfOperationError("The operation finished in error with code %s: %s" % (
-                  created_operation["error"]["code"],
-                  created_operation["error"]["message"]))
+            raise SdfOperationError(
+                "The operation finished in error with code %s: %s"
+                % (created_operation["error"]["code"], created_operation["error"]["message"])
+            )
         self.download_sdf(created_operation)
         unzip(f"{self.BASE}/{self.ARCHIVE_NAME}.zip", output_path=self.BASE)
 
         # We chain operation if many file_types were to be provided.
-        return chain(
-            *[
-                sdf_to_njson_generator(f"{self.BASE}/{file_name}.csv")
-                for file_name in self.file_names
-            ]
-        )
+        return chain(*[sdf_to_njson_generator(f"{self.BASE}/{file_name}.csv") for file_name in self.file_names])
+
+    def __get_creatives(self):
```

Comment:
Je ne vois pas bien comment ajouter des tests pertinents là-dessus non plus sans que ce soit overkill, étant donné qu'on ne fait que récupérer la réponse brute de l'API... Je pense qu'on peut laisser ça comme ça 😊 

Seule petite chose: est-ce qu'on harmoniserait pas le nom des méthodes non-exposées (comme tu me l'avais suggéré sur le reader The Trade Desk, en les précédant d'underscores) ? C'est le cas de la méthode `__get_creatives` que tu as ajoutée, mais pas des autres.

Reply:
J'aurais aimé tester la logique de pagination, de continuer si on a un token, sinon d'arrêter etc. J'ai fait des tests manuels mais ça aurait été top de pouvoir en faire de manière automatique.

Noté pour les méthodes private, je change ça.

Reply:
Je vois, c'est une bonne idée en effet, juste pas ultra-évidente à mettre en pratique car il faut mocker l'enchainement de méthodes appliquées au client... A voir si tu as le temps, sinon on commence sans 😊